overview.md

```diff
@@ -0,0 +1,180 @@
+
+The Collection specification extends the Catalog spec, so every Collection can serve as a catalog, 
+requiring the same fields. But it has a number of additional fields: license, extent (spatial and 
+temporal), providers, keywords and summaries. So every Item in the Collection must share those fields, 
+and if Items are in a Collection they must link back to their Collection, so clients can easily find 
+fields like the license. 
+
+But what *should* go in a Collection, versus just in a Catalog?  A collection will generally consist of
+a set of assets that are defined with the same properties and share higher level metadata. In the 
+satellite world these would typically all come from the same sensor or constellation. It corresponds
+directly to what others call a 'dataset series' (ESA, ISO 19115), 'collection' (CNES, NASA), and 
+'dataset (JAXA, DCAT)'. So if all your items have the same properties, they probably belong in 
+the same Collection. But the construct is deliberately flexible, as there may be good reasons
+to 'break' the recommendation. 
+
+Catalogs in turn are used for two main things:
+
+* Split overly large collections into groups
+* Group collections into a catalog of collections (i.e. as entry point).
+
+The first case allows users to browse down into the items of big collections. A collection like
+Landsat usually would start with path and row Catalogs to group by geography, and then year, 
+month and day groups to enable deeper. [Dynamic catalogs](best-practices.md#dynamic-catalogs) can
+provide multiple grouping paths, serving as a sort of faceted search.
+
+The second case is used when one wants to represent diverse data in a single place. If an organization
+has an internal catalog with Landsat 8, Sentinel 2, NAIP data and several commercial imagery providers
+then they'd have a root catalog that would link to a number of different collections. 
+
+So in conclusion it's best to use collections for what you want user to find as starting point, and then
+catalogs are just for structuring and grouping the data.
+
+## Catalog Overview
+
+There are two required element types of a Catalog: Catalog and Item. A STAC Catalog
+points to [STAC Items](item-spec/README.md), or to other STAC catalogs. It provides a simple
+linking structure that can be used recursively so that many Items can be included in 
+a single Catalog, organized however the implementor desires. 
+
+STAC makes no formal distinction between a "root" catalog and the "child" catalogs. A root catalog
+is simply a top-most catalog (which has no parent). A nested catalog structure is useful (and
+recommended) for breaking up massive numbers of catalog items into logical groupings. For example,
+it might make sense to organize a catalog by date (year, month, day), or geography (continent,
+country, state/prov). Any scheme may be used, but it's considered a best practice to keep the size
+of each catalog under a megabyte.
```

Comment:
So should this better be in best practices?

Reply:
That's fair, will look into it.

Reply:
moved